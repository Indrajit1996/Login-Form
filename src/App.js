import React from 'react';
import Login from './containers/Login.js'
import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
          <p>Login Page</p>
        <Login />
      </header>
    </div>
  );
}

export default App;