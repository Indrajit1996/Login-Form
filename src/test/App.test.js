import React from 'react';
import { render } from '@testing-library/react';
import App, { Counter } from '../App';
import { create } from "react-test-renderer";

it('renders Login Page', () => {
  const { getByText } = render(<App />);
  const linkElement = getByText(/Login Page/i);
  expect(linkElement).toBeInTheDocument();
});

describe("App component", () => {
  test("Matches the snapshot", () => {
    const app = create(<App />);
    expect(app.toJSON()).toMatchSnapshot();
  });
});


