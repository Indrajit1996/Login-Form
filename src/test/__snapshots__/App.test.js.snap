// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Matches the snapshot 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <p>
      Login Page
    </p>
    <div
      className="container"
    >
      <div
        className="center card"
      >
        <h3>
          <div
            className="logo-box"
          >
            <img
              alt="logo"
              src="https://s3.ap-south-1.amazonaws.com/tpng-images/random/original/6be52f0b-56bf-4fdc-abd3-0bec9e58610c.png"
            />
          </div>
          <div
            className="sign-in"
          >
            Sign In 
          </div>
          <div
            className="title"
          >
            Use your Healthifyme Account 
          </div>
        </h3>
        <label
          htmlFor="email"
        />
        <input
          className="input"
          id="email"
          name="email"
          onChange={[Function]}
          placeholder="Enter Your Email"
          style={null}
          type="email"
          value=""
        />
        
        <label
          htmlFor="password"
        />
        <input
          className="input"
          id="password"
          name="password"
          onChange={[Function]}
          placeholder="Enter Your Password"
          style={null}
          type="password"
          value=""
        />
        
        <button
          className="push_button red isDisabled"
          disabled={true}
          onClick={[Function]}
          type="submit"
        >
          Login
        </button>
      </div>
    </div>
  </header>
</div>
`;

exports[`Button component Matches the snapshot 1`] = `<button />`;
